src/atmosphere.jl

```diff
@@ -51,6 +51,43 @@ function Base.show(io::IO, m::MIME"text/plain", atm::A) where A <: ModelAtmosphe
     print(io, "$(A) with $(length(atm.layers)) layers")
 end
 
+#TODO add tests
+"""   
+    tau_5000s(atm::ModelAtmosphere) = [l.tau_5000 for l in atm.layers]
+
+This is a convienince functions for making plots, etc.  Note that it doesn't access quantities in a 
+memory-efficient order.
+"""
+tau_5000s(atm::ModelAtmosphere) = [l.tau_5000 for l in atm.layers]
+"""
+   zs(atm::ModelAtmosphere) = [l.z for l in atm.layers]
+
+This is a convienince functions for making plots, etc.  Note that it doesn't access quantities in a 
+memory-efficient order.
+"""
+zs(atm::ModelAtmosphere) = [l.z for l in atm.layers]
+"""
+   temps(atm::ModelAtmosphere) = [l.temp for l in atm.layers]
+
+This is a convienince functions for making plots, etc.  Note that it doesn't access quantities in a 
+memory-efficient order.
+"""
+temps(atm::ModelAtmosphere) = [l.temp for l in atm.layers]
+"""
+   electron_number_densities(atm::ModelAtmosphere) = [l.electron_number_density for l in atm.layers]
+
+This is a convienince functions for making plots, etc.  Note that it doesn't access quantities in a 
+memory-efficient order.
+"""
+electron_number_densities(atm::ModelAtmosphere) = [l.electron_number_density for l in atm.layers]
+"""
+    number_densities(atm::ModelAtmosphere) = [l.number_density for l in atm.layers]
+
+This is a convienince functions for making plots, etc.  Note that it doesn't access quantities in a 
+memory-efficient order.
+"""
+number_densities(atm::ModelAtmosphere) = [l.number_density for l in atm.layers]
+
```

Comment:
One thought that may be worth considering for a future PR. What if instead of making 5 helper functions that each handled one quantity, you made a single function that looked something like the following:

```julia
function atm_to_dict(atm::ModelAtmosphere)
    map([:tau_5000, :z, :temp, :electron_number_density, :number_density]) do prop
        String(prop) => [getfield(l,prop) for l in atm.layers] 
    end |> Dict
end
```

Since it's just a convenience function it's okay for this to be a little slow. Plus, you would be able to avoid adding these generic function names to namespace.

With that said, if you prefer your current solution, that's fine with me.

Reply:
I think this leaves a bit to be desired in just in that it requires thinking about a new object.  In general, I think that the singular form of a noun is almost always sensible as a variable name when the plural is.  I do actually get the hesitancy, and using `get_zs` type names might be a good alternative.

Btw, these changes were introduced in #133, but we can just merge this and delete that branch.

Reply:
The more I ponder this, the more I'm leaning towards `get_temps` styles names.  Unless you object, I'm going to do that.

Reply:
I've done it.  Lmk if you hate it.

Reply:
That's a good idea. I prefer that over what you currently have.

One last alternative (but again, feel free to ignore - I'm just bikeshedding here):
```julia
function query_atm_prop(atm::ModelAtmosphere, prop)
    prop_symbol = Symbol(prop)
    @assert prop_symbol in fieldnames(ModelAtmosphere)
    getfield.(atm.layers, prop_symbol)
end
```

Reply:
What is the reasoning for something like this over what I have?  Shorter implementation?

Reply:
Shorter implementation, fewer names in the namespace... It all comes down to personal preference. There's really no need to change it -- what you have definitely works. I'm going to hit merge